app.js

```diff
@@ -1,3 +1,72 @@
+const projects = [
+  {
+    id: 1,
+    name: 'Multi-Post Stories Gain+Glory',
+    image: './assets/Img Plaholder.png',
+    description: `Lorem ipsum dolor sit amet consectetur, adipisicing elit. Cupiditate, sunt ea, saepe
+    aspernatur reiciendis doloribus deserunt perspiciatis eaque porro tempore consequatur voluptatibus nam veniam
+    officia, placeat praesentium dignissimos alias est?`,
+    technologies: ['Ruby on Rails', 'CSS', 'Javascript'],
+    liveVersion: 'https://majdalkilany.github.io/Portfolio-/',
+    sourceCode: 'https://github.com/majdalkilany/Portfolio-',
+  },
```

Comment:
- [ ] As suggested by the previous reviewer, kindly adjust the buttons to align properly to the left to in par with the paragraph. This is necessary to give a better user experience and shows attention to detail in your work 👍 
- You can consider adjusting using flexbox with a flex-start to make it fit 👍 

## View:
![image](https://github.com/majdalkilany/Portfolio-/assets/50990925/ba2811a5-9f26-4d5b-a2e5-6a5d650ef5f4)